newsfragments/3752.bugfix.rst

```diff
@@ -0,0 +1 @@
+Fix the handling of special characters in the Send-To field when sending invitation email
```

Comment:
"Fix user invitation email sending error when the greeter's name contains special characters"